build-bin/dist_to_gh_pages

```diff
@@ -15,26 +15,37 @@
 
 set -ue
 
+ORIGIN_BRANCH='master'
+TARGET_BRANCH='gh-pages'
+
 # Update gh-pages
-git fetch origin gh-pages:gh-pages
-git checkout gh-pages
+git fetch origin $TARGET_BRANCH:$TARGET_BRANCH
+git checkout $TARGET_BRANCH
+
+
+
+git checkout -qf $TARGET_BRANCH
+
+# Remove everything except .git
+rm -rf *
+
+git checkout $ORIGIN_BRANCH docs
 
-git checkout -qf gh-pages
+cd docs
+pnpm i
+pnpm run build
 
-# TODO: these files can be in a list for convenience.
```

Comment:
maybe this is why you were adding this in the vite-config? This deserves a comment there to avoid me re-asking the same questions!

something like?

```
// these files were formerly checked out from master in build-bin/idl_to_gh_pages.
// We now package them directly when making the openapi dist
```